lib/triangles.js

```diff
@@ -0,0 +1,33 @@
+var pry = require('pryjs');
+// eval(pry.it);
+
+function validity_checker(a, b, c) {
```

Comment:
be sure to follow JS convention when naming variables/functions - these should be camel case instead of snake case 👍